config/rbac/rbac_role.yaml

```diff
@@ -89,7 +89,6 @@ rules:
   - cluster.k8s.io
   resources:
   - machines
-  - machines/status
```

Comment:
Seems something has changed here, I did regenerate after updating Go, but I think that's unrelated